Sources/SmileID/Classes/Helpers/LocalStorage.swift

```diff
@@ -2,7 +2,9 @@ import Foundation
 import Zip
 
 public class LocalStorage {
-    private static let defaultFolderName = "sid_jobs"
+    private static let defaultFolderName = "SmileID"
+    private static let pendingFolderName = "unsubmitted"
+    private static let completedFolderName = "submitted"
```

Comment:
Update variable names to reflect new naming scheme

```suggestion
    private static let unsubmittedFolderName = "unsubmitted"
    private static let submittedFolderName = "submitted"
```

Reply:
good catch, fixed